pkg/antctl/raw/set/flowaggregator/command.go

```diff
@@ -133,6 +141,8 @@ func updateRunE(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	configMap.Data = make(map[string]string)
```

Comment:
I think this line should be removed, otherwise, you will override all existing data with `flow-aggregator.conf` only.

Reply:
actually the configMap.Data was <nil> for the fakeConfigMap used, and if try to assign the value to nil it will generate an error "assignment to entry in nil map', so we need to first make the map and then proceed. But for safety and taking above case that you stated in mind, I can enclose this statement inside an if condition, and include the check if the data is nil then only this statement will be executed.